content/en/docs/IV/modules/ecg/waveform_note_links.md

```diff
@@ -0,0 +1,44 @@
+---
+title: "Waveform Note Links Table"
+linktitle: "waveform_note_links"
+date: 2023-10-02
+weight: 30
+description: >
+  A table which provides links between waveforms and available cardiologist notes.
+---
+
+{{% pageinfo %}}
+Please note that the free-text cardiologist notes for the ECG module will be publicly released in the MIMIC-IV-Note module at a later date. An initial set of these notes are available to our consortium members only, at this time.
```

Comment:
I'm not "our consortium" will mean much to people?

Reply:
@tompollard I agree, I've removed the mention of our consortium.